service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -102,12 +103,30 @@ public ResponseEntity<RunSetStateResponse> postRunSet(RunSetRequest request) {
     Map<String, Object> params =
         InputGenerator.buildInputs(request.getWorkflowInputDefinitions(), recordResponse);
 
+    UUID runId = UUID.randomUUID();
+    String dataTableRowId = request.getWdsRecords().getRecordIds().get(0);
+
     // Submit the workflow and get its ID:
     RunId workflowResponse;
+    String errorMessage;
```

Comment:
Since it's only used inside the `catch` I think you could move the variable declaration down to line 116, ie at the same point it gets a value set.